src/Infrastructure.VS.UnitTests/SolutionInfoProviderTests.cs

```diff
@@ -64,19 +77,64 @@ public async Task Get_ReturnsExpectedValue(string solutionNameToReturn)
         }
 
         [TestMethod]
-        public async Task Get_ServiceCalledOnBackgroundThread()
+        [DataRow(null)]
+        [DataRow("a value")]
+        public void Get_ReturnsExpectedValue(string solutionNameToReturn)
```

Comment:
Copy of the `GetAsync_ReturnsExpectedValue` test, modified to call the synchronous method.